### PR TITLE
Adding best practices section

### PR DESCRIPTION
**Pull request by emanuelhaine**
As I talked to Igor, I am opening this PR so I can start helping on the issus below:

https://github.com/ZOSOpenTools/meta/issues/830
https://github.com/ZOSOpenTools/meta/issues/831

This PR introduces the Best Practics section to the zopen docs
zopen_doc

### CODE REVIEW

**Comment by IgorTodorovskiIBM** [comment]
@emanuelhaine I see additional changes listed. I think you need to sync your fork/branch with the latest in meta:main

**Comment by emanuelhaine** [comment]
@IgorTodorovskiIBM ,

I resolved the conflicts. Let me know if it is good now.

File: docs/_sidebar.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
     - [Newly Released Packages](/newly_released.md)
     - [Package Vulnerabilities](/Vulnerabilities.md)
     - [Current Status](/Progress.md)
+  - [Best Practices](/Guides/section.md)
+    - zopen
```
Comment (IgorTodorovskiIBM) [inline docs/_sidebar.md:13]: Are you able to create a stub page for these?

**Comment by emanuelhaine** [comment]
@IgorTodorovskiIBM 

It is done. Let me know if you need anything else.

**Comment by IgorTodorovskiIBM** [comment]
Thanks, next step would be adding the content but I'm ok with this for now. Perhaps add "(coming soon)" or or similar in the content for now?

**Comment by emanuelhaine** [comment]
@IgorTodorovskiIBM , I added the  "(coming soon)"  to new pages that I created

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**